src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+@import "./utils/variables.scss";
```

Comment:
there is no need to use @import in all files. It's enough to use it only in the main.scss

Reply:
file with variables from this file, all the code breaks down